velox/common/config/Config.h

```diff
@@ -158,4 +158,18 @@ class ConfigBase {
   const bool mutable_;
 };
 
+class GlobalConfig {
+ public:
+  template <typename T>
+  static T get(const std::string& key);
```

Comment:
String keys and type casting at runtime are not safe or efficient for this case.  Looks like we just need a collection of global variables.  Or a global `struct` with keys and types known at compile time.